app/assets/javascripts/stores/user_store.js

```diff
@@ -5,7 +5,8 @@ const UserStore = new StockStore({
   sortAsc: true,
   descKeys: {
     character_sum_ms: true,
-    character_sum_us: true
+    character_sum_us: true,
+    character_sum_ds: true
```

Comment:
Likewise: `character_sum_draft`.

Reply:
Should I do the same changes in locales files also? @ragesoss

Reply:
No. These come from the `CoursesUsers` model, but they aren't user-facing labels. Changing them in these two places will fix the sorting.

The labels are actually pretty bad, and changing them to something clearer would be helpful, that's independent of this fix.

Reply:
I can try to change the labels if you feel that would be better.